Update service entry management in SC (#21840)
DataDog/documentation: content/en/service_catalog/setup.md

Datadog automatically discovers the dependencies of instrumented services, including databases or third-party APIs, even if the dependency hasn't been instrumented. The Service Catalog lists these as separate entries. To differentiate auto-detected components from instrumented services, you can request access to the private beta for inferred services.
{{< /callout >}}

## Create user-defined services 

> You can bring your own services to Service Catalog. You can either manually add them by creating Service Definitions through API or GitHub integration or [import][19] them from existing sources like ServiceNow or Backstage. These services are by default not associated with any Datadog telemetry, you may link telemetries from Datadog or external sources manually through `service.datadog.yaml` files. 

Reviewer: ```suggestion
To add your own services to Service Catalog, you can either manually add them by creating Service Definitions through the API or GitHub integration or [import](#import-data-from-other-sources) them from existing sources like ServiceNow or Backstage. These services are by default not associated with any Datadog telemetry, but you can link telemetries from Datadog or external sources manually using `service.datadog.yaml` files. 
```

You can populate your Datadog Service Catalog with services from your ServiceNow CMDB by using the Service Ingestion feature in the [Datadog-ServiceNow integration][16].

> ## Manual Service Discovery through other Datadog Telemetries

Reviewer: ```suggestion
## Manual Service Discovery through other Datadog telemetries
```

To discover additional services through existing Datadog telemetry such as infrastructure metrics, navigate to the [**Setup & Config** tab][3] on the top of the page and click on the **Import Entries** tab. You can import services from other Datadog telemetry containing the `DD_SERVICE` [tag][5].

{{< img src="tracing/service_catalog/import_entries.png" alt="Import Entries tab in the Service Catalog setup and configuration section" style="width:90%;" >}}

> Once you have imported some entries, they appear in the **Explore** tab. Entries may expire unless you add metadata such as the owner or contacts by [using the API][1] or the [GitHub integration][6].

Reviewer: ```suggestion
After you have imported some entries, they appear in the **Explore** tab. Entries may expire unless you add metadata such as the owner or contacts by [using the API][1] or the [GitHub integration][6].
```

[16]: https://docs.datadoghq.com/integrations/servicenow/#service-ingestion
[17]: https://docs.datadoghq.com/universal_service_monitoring/
[18]: https://docs.datadoghq.com/tracing/

> [19]: /service_catalog/setup#import-data-from-other-sources

Reviewer: Added an anchor link for this instead, since it's on the same page.

```suggestion
```
